services/app-web/src/components/Form/FieldCheckbox/FieldCheckbox.test.tsx

```diff
@@ -28,6 +28,8 @@ describe('FieldCheckbox component', () => {
                 name="managedCareEntity"
                 id="mco"
                 label="Managed Care Organization (MCO)"
+                heading="Managed Care entities"
+                parent_component_heading="Contract Details Form"
```

Comment:
Please make "Contract Details Form" a `const` in case the name of form changes its easy to update everywhere